### PR TITLE
[TCSACR-584][NUI] Window.Instance is deprecated and Default is newly added

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Window.Instance is deprecated and Default is newly added

### API Changes ###
https://jira.sec.samsung.net/browse/TCSACR-584

### CODE REVIEW

File: src/Tizen.NUI/src/public/Window/Window.cs [hunk 1]
```diff
@@ -424,6 +424,13 @@ public enum ResizeDirection
         /// <since_tizen> 3 </since_tizen>
         public static Window Instance { get; internal set; }
 
+
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/Window/Window.cs:427]: un

File: src/Tizen.NUI/src/public/Window/Window.cs [hunk 2]
```diff
@@ -424,6 +424,13 @@ public enum ResizeDirection
...
+        /// <summary>
+        /// Gets the default window.
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/Window/Window.cs:429]: you might consider adding sth more, like "Gets the default window - the initial window of the application.", maybe terms like "main window", "top window" do not fit or maybe the do. I just would prefer to have some explanation added about how it is default, which one is it, does it change, it is always available? On the other hand most users will find that they should just use this "Window.Default" and that will be all they need to know.

also, the extra empty line 427 is not needed

Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Window/Window.cs:427]: @wiertel 
Thank you for reviewing. I have changed it according to your comments.

**Review by wiertel — APPROVED** [review]

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**